test/mailers/queued_email_test.rb

```diff
@@ -231,13 +231,16 @@ def test_author_request_email
   def test_verify_api_key_email
     key = api_keys(:marys_api_key)
 
+    # Dick is creating an API for Mary at Mary's request.
+    # The email is from Dick to Mary, the "user" is Mary, the "other_user"
+    # is Dick.
     QueuedEmail::VerifyAPIKey.create_email(mary, dick, key)
     assert_email(0,
                  flavor: "QueuedEmail::VerifyAPIKey",
-                 from: mary,
-                 to: dick,
+                 from: dick,
+                 to: mary,
                  user: mary.id,
-                 other_user: dick.id,
+                 other_user: disk.id,
```

Comment:
@pellaea Why`disk` and not `dick`? 
And shall we go ahead and change this param now to `app_user` while we are thinking about it?

Reply:
Oh right the `app_user` param is for `user`, not `other_user`. Anyway... let's get those var names as clear as we can while we have a bit of light on the subject...

Reply:
lol. thanks. i'll handle the var renaming then, if i can keep it straight!